ui/app/manager/src/pages/page-alarms.ts

```diff
@@ -0,0 +1,1133 @@
+                                        type="${InputType.BUTTON}"
+                                        icon="delete"
+                                        @or-mwc-input-changed="${() => (this._deleteAlarms())}"
+                                ></or-mwc-input>
+                            </div>
+                            <hr class="${this._selected?.length > 0 ? "" : "hidden"}">
+                            <div class="${this.creationState || this.alarm ? "hidden" : "controls-left"}">
+                                <or-mwc-input .type="${InputType.CHECKBOX}" id="assign-check"
+                                              ?disabled="${disabled}" .label="${i18next.t("alarm.assignedToMe")}"
+                                              @or-mwc-input-changed="${(evt: OrInputChangedEvent) => this._onAssignCheckChanged(evt.detail.value)}"
+                                              .value="${this.assign}"></or-mwc-input>
+                                <or-mwc-input .type="${InputType.SELECT}" id="severity-select" comfortable
+                                              ?disabled="${disabled}" .label="${i18next.t("alarm.severity")}"
+                                              @or-mwc-input-changed="${(evt: OrInputChangedEvent) => this._onSeverityChanged(evt.detail.value)}"
+                                              .value="${this._getSeverityOptions().filter((obj) => obj.value === this.severity || !this.status ? obj.value === 'All' : '').map((obj) => obj.label)[0]}"
+                                              .options="${this._getSeverityOptions().map( s => s.label)}"></or-mwc-input>
+                                <or-mwc-input .type="${InputType.SELECT}" id="status-select" comfortable
+                                              ?disabled="${disabled}" .label="${i18next.t("alarm.status")}"
+                                              @or-mwc-input-changed="${(evt: OrInputChangedEvent) => this._onStatusChanged(evt.detail.value)}"
+                                              .value="${this._getStatusOptions().filter((obj) => obj.value === this.status || this.allActive ? obj.value === 'All-active' : '' || !this.status ? obj.value === 'All' : '').map((obj) => obj.label)[0]}"
+                                              .options="${this._getStatusOptions().map( s => s.label)}"></or-mwc-input>
+
+                            </div>
+                            <div class="${this.creationState || this.alarm || assignOnly || readonly ? "hidden" : "panel-title"}"
+                                 style="justify-content: flex-end;">
+                                <or-mwc-input
+                                        raised
+                                        style="padding: 0px 10px; margin: 0;"
+                                        type="${InputType.BUTTON}"
+                                        icon="plus"
+                                        label="${i18next.t("add")} ${i18next.t("alarm.")}"
```

Comment:
Would recommend to just add a new entry in the translation file with `alarm.addAlarm`, as different languages use these combinations of words differently.